improver_tests/utilities/test_CalculateClimateAnomalies.py

```diff
@@ -92,38 +92,46 @@ def std_cube(validity_time, time_bounds):
         data=data,
         time=validity_time["mean_and_std"],
         time_bounds=time_bounds["mean_and_std"],
-        units="K",
+        units="1",
```

Comment:
```suggestion
        units="K",
```
This was correct before, as described in a previous comment.